lib/error.g

```diff
@@ -349,16 +348,8 @@ BIND_GLOBAL("ErrorInner", function(options, earlyMessage)
     else
         prompt := "brk> ";
     fi;
-    shellOut := "*errout*";
-    shellIn := "*errin*";
```

Comment:
Ah, I am stupid. Of course we must use errout/errin here. So I need to add back one argument, something like `BOOL isBreakPrompt` or so.